Server/API/sse/getSSE.go

```diff
@@ -0,0 +1,34 @@
+package sse
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	entities "ishtaloo.io/Entities"
+)
+
+func getSSE(c echo.Context, sseChannel *entities.SSEChannel) error {
+	c.Response().Header().Set(echo.HeaderContentType, "text/event-stream")
+	c.Response().Header().Set("Cache-Control", "no-cache")
+	c.Response().Header().Set("Connection", "keep-alive")
+	c.Response().WriteHeader(http.StatusOK)
```

Comment:
move those lines to a separate function, something like setSSEHeaders(c)

Reply:
Will do